cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -99,6 +99,10 @@ features:
     sendEventsButtonLabel: Send
     sendEventsClickLabel: Click to input events
     sendEventsFailed: Error - Send Events Failed
+    startConfirmLabel: Start
+    stopConfirmLabel: Stop
+    startConfirmation: Start *_{entityId}_*?
```

Comment:
This definitely needs to be more descriptive. `Are you sure you want to start the program: **ProgramName**` would be more appropriate.